### PR TITLE
Remove App Version and collections methods

### PR DESCRIPTION
**Pull request by illyaV**
- change API calls
- change handlers of API calls response
- fix examples

### CODE REVIEW

File: examples/data-service/js/app.js [hunk 1]
```diff
@@ -14,17 +14,16 @@ jQuery(function( $ ) {
...
-        VERSION : 'v1'
+        SECRET_KEY : ''
     };
 
 	var App = {
 		init: function() {
             //Backendless: initialize API
-            if( !Defaults.APPLICATION_ID || !Defaults.SECRET_KEY || !Defaults.VERSION )
+            if( !Defaults.APPLICATION_ID || !Defaults.SECRET_KEY)
                 alert( "Missing application ID and secret key arguments. Login to Backendless Console, select your app and get the ID and key from the Manage > App Settings screen. Copy/paste the values into the Backendless.initApp call located in app.js" );
```
Comment (stanislav-grin) [inline examples/data-service/js/app.js]: 'Missing application ID _**or**_ secret key ...' I think would be correct.

File: libs/backendless.js [hunk 2]
```diff
@@ -2165,32 +2111,36 @@
...
-                        data.collection.data[i] = geoObject;
+
+                        resultGeoObject.categories = geoObject.categories;
+                        resultGeoObject.latitude = geoObject.latitude;
+                        resultGeoObject.longitude = geoObject.longitude;
+                        resultGeoObject.metadata = geoObject.metadata;
+                        resultGeoObject.objectId = geoObject.objectId;
+                        resultGeoObject.distance = geoObject.distance;
+
+                        geoCollection.push(resultGeoObject);
```
Comment (vengrov) [inline libs/backendless.js]:
```js
var geoObject = data.collection[i]
var isCluster = geoObject.hasOwnProperty('totalPoints');
var GeoItemType = isCluster ? GeoCluster : GeoPoint
var geoItem = new GeoItemType(geoObject)

geoItem.geoQuery = query; //it seems ok to apply this field to point as well
geoCollection.push(geoItem)
```

File: examples/data-service/js/app.js (hunk 1 above)
Comment (illyaV) [inline examples/data-service/js/app.js]: fixed

File: libs/backendless.js (hunk 2 above)
Comment (illyaV) [inline libs/backendless.js]: done

**Pull request merged**